.circleci/config.yml

```diff
@@ -26,6 +26,14 @@ jobs:
       - sfdx/install:
           version: latest
 
+  integration-install_cli_npm_newer_node:
+    docker:
+      - image: cimg/node:14.7
+    working_directory: ~/project
+    steps:
+      - sfdx/install:
+          version: latest
```

Comment:
Could you see what happens here if you also do 

```
- sfdx/auth:
  defaultusername: gabriel@circleci.pe
- run:
    name: Check Auth List
    command: sfdx force:auth:list
```

like is done in the integration-install_authenticate test below but for cimg/node:14.7?

I'm not sure if I open a pull request  that it will run the external PR since this contains sensitive information.